doc/rst/drm.1.rst

```diff
@@ -58,6 +58,13 @@ OPTIONS
    them. This is useful to check list of items satisfying --exclude or
    --match options before actually deleting anything.
 
+.. option:: --aggressive
+
+   This option will delete files during the walk phase, and then
+   delete directories by level after the walk in drm. Using this
+   option means that you cannot use the --dryrun option in the
+   same program run as the --aggressive option.
+
```

Comment:
Let's just simplify the second sentence to "You cannot use this option with --dryrun."

Reply:
@adammoody ok it is updated.